### DISCUSSION TITLE
problems with artemis virtual gamepad

### DISCUSSION THREAD

**Discussion by dsanttos12** [ClassicOldSong/moonlight-android#61] [General]
The virtual gamepad has a strange layout and is not working with any PC game. 
![Screenshot_20241122_161853_Artemis](https://github.com/user-attachments/assets/1242113b-01eb-4b40-8708-ad82e6d530bd)

**Comment by ClassicOldSong** [comment]
This is the standard layout. You can modify the layout by tapping the gear icon on the top left.

Check your PC if the vigembus driver is installed correctly, and what's the current controller it's emulating.

**Reply by dsanttos12** [reply]
I want to remove this switch layout and put the Xbox layout
